src/NFTEDA.sol

```diff
@@ -124,8 +141,16 @@ abstract contract NFTEDA {
         emit EndAuction(id, price);
     }
 
+    /// @notice Sets the time at which the auction was started
+    /// @dev abstracted to a function to allow developer some freedom with how to store auction state
+    /// @param id The id of the auction
     function _setAuctionStartTime(uint256 id) internal virtual;
 
+    /// @notice Clears all stored state for the auction
+    /// @dev abstracted to a function to allow developer some freedom with how to store auction state
+    /// @param id The id of the auction
+    function _clearAuctionState(uint256 id) internal virtual;
```

Comment:
same implementation in both subclasses, worth implementing here?

Reply:
they're different!